Bug: tsconfig.build.json missing in published npm bundle

Upon installing version `3.5.4` of the new `@apollo/server-plugin-operation-registry` package, we get an error:

```
error TS5083: Cannot read file '/<directory>/node_modules/@apollo/server-plugin-operation-registry/tsconfig.build.json'.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @apollo/server-plugin-operation-registry@3.5.4 build: `tsc --build tsconfig.build.json`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @apollo/server-plugin-operation-registry@3.5.4 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

This seems to be a problem with the `postinstall` npm script, which runs the `build` command referencing the file `tsconfig.build.json`. But in the downloaded bundle, that file seems to be missing entirely:

<img width="382" alt="Screen Shot 2022-03-28 at 1 05 38 PM" src="https://user-images.githubusercontent.com/1036549/160450491-0adc1371-c9cf-4261-8ca8-b4deeecac483.png">

I think it's likely not being included during the publish step? But wasn't sure how to check that.

--- a/.changeset/poor-paws-grin.md
+++ b/.changeset/poor-paws-grin.md
@@ -1,0 +1,5 @@
+---
+'@apollo/server-plugin-operation-registry': patch
+---
+
+Add tsconfig files to published output

--- a/.npmignore
+++ b/.npmignore
@@ -5,6 +5,6 @@ src/**/__tests__/**
 dist/**/__tests__/**
 !package.json
 !README.md
-tsconfig.base.json
-tsconfig.json
-tsconfig.build.json
+!tsconfig.base.json
+!tsconfig.json
+!tsconfig.build.json

--- a/.npmignore
+++ b/.npmignore
@@ -5,3 +5,6 @@ src/**/__tests__/**
 dist/**/__tests__/**
 !package.json
 !README.md
+tsconfig.base.json
+tsconfig.json
+tsconfig.build.json

